src/frontend/src/pages/landingpage/faq/faqlist.tsx

```diff
@@ -0,0 +1,30 @@
+import type { PropsWithChildren } from "react";
+
+
+const Link = ({href, children}: PropsWithChildren<{ href: string }>) => <a className="text-blue-700 dark:text-blue-400 hover:underline" href={href}>{children}</a>;
+
+
+export const FAQList: Array<{question: string, answer: string | JSX.Element}> = [
+    {
+        question: "What is the difference to other git analysis sites?",
+        answer: "We offer analysis over all your repositories and not just one repository at a time",
```

Comment:
```suggestion
        answer: "Instead of analyzing only one repository, we can analyze an entire workspace. A feature, that many other products lack.",
```